modProcess.bas

```diff
@@ -1019,7 +1019,7 @@ Sub ProcessString(Index As Long, PacketID As Long, St As String)
 
             Case 6    'Say
                 If Not .Flag(40) > 0 And Not .Flag(41) > 0 And .IsDead = False Then
-                    .FloodTimer = .FloodTimer + 1000
+                    .FloodTimer = .FloodTimer + 500

```

Comment:
Could we convert the 500 to a variable, so it could be set in options?

Reply:
Yes thats possible. However, each method of chat has a different Floodtimer input, but this could be changed to just one set amount for all.

Reply:
For example, Broadcast has a bigger Floodtimer input meaning that if you spam /b then you're more likely to get booted and squelched than if you spam map chat etc

Reply:
Is this something that needs to be changed before the merge, or can it be done after?